### DISCUSSION TITLE
I removed Material Shell and my dock is still gone

### DISCUSSION THREAD

**Discussion by summerfr33ze** [material-shell/material-shell#881] [General]
I was hoping Material Shell was something I could remove easily if I had issues with bugs. I had a few and decided that something simpler would work better for me. The problem is that the uninstall section isn't telling me how to get my dock back.

**Comment by PapyElGringo** [comment]
Maybe it's just an extension to reenable ! Check your extensions

**Comment by summerfr33ze** [comment]
Yeah I ended up being able to fix it by googling what could cause the dock
to disappear. I think in the uninstall portion of your github page you
should just tell people how to re-enable the dock so they don't  have to
spend 20 minutes trying to figure it out.

**Comment by paulovictor237** [comment]
# Reset GNOME Desktop Settings
full reset will fix all your problems
```bash
dconf reset -f /org/gnome/
```
In case you what to make a backup of all your settings before install material shell:
```bash
cd ~/Downloads
dconf dump / > full-backup # full backup
dconf load / < full-backup # full restore
```
